### PR TITLE
Remove security cameras for real

### PR DESCRIPTION
**Pull request by optimumtact**
For real this time

### CODE REVIEW

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -2,7 +2,7 @@
 	name = "helmet"
 	desc = "Standard Security gear. Protects the head from impacts."
 	icon_state = "helmet"
-	flags = HEADBANGPROTECT
+	flags = HEADCOVERSEYES | HEADBANGPROTECT
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: WRONG.
flags_covers for HEADCOVERSEYES

File: code/modules/clothing/head/helmet.dm [hunk 2]
```diff
@@ -11,29 +11,15 @@
 	heat_protection = HEAD
 	max_heat_protection_temperature = HELMET_MAX_TEMP_PROTECT
 	strip_delay = 60
-	burn_state = -1 //Won't burn in fires
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: you're reverting this

File: code/modules/clothing/head/helmet.dm [hunk 3]
```diff
@@ -128,7 +114,7 @@
 /obj/item/clothing/head/helmet/roman
 	name = "roman helmet"
 	desc = "An ancient helmet made of bronze and leather."
-	flags_cover = HEADCOVERSEYES
+	flags = HEADCOVERSEYES
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: wrong

File: code/modules/clothing/head/helmet.dm [hunk 4]
```diff
@@ -144,16 +130,15 @@
 	name = "gladiator helmet"
 	desc = "Ave, Imperator, morituri te salutant."
 	icon_state = "gladiator"
-	flags = BLOCKHAIR
+	flags = HEADCOVERSEYES|BLOCKHAIR
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: wrong

File: code/modules/clothing/head/helmet.dm [hunk 5]
```diff
@@ -144,16 +130,15 @@
...
 	item_state = "gladiator"
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES
-	flags_cover = HEADCOVERSEYES
 
 /obj/item/clothing/head/helmet/redtaghelm
 	name = "red laser tag helmet"
 	desc = "They have chosen their own end."
 	icon_state = "redtaghelm"
-	flags_cover = HEADCOVERSEYES
+	flags = HEADCOVERSEYES
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: wrong

File: code/modules/clothing/head/helmet.dm [hunk 6]
```diff
@@ -163,7 +148,7 @@
 	name = "blue laser tag helmet"
 	desc = "They'll need more men."
 	icon_state = "bluetaghelm"
-	flags_cover = HEADCOVERSEYES
+	flags = HEADCOVERSEYES
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: wrong

File: code/modules/clothing/head/helmet.dm (hunk 2 above)
Comment (optimumtact) [inline code/modules/clothing/head/helmet.dm]: good catch

File: code/modules/clothing/head/helmet.dm [hunk 7]
```diff
@@ -51,14 +37,14 @@
 	toggle_message = "You pull the visor down on"
 	alt_toggle_message = "You push the visor up on"
 	can_toggle = 1
-	flags = HEADBANGPROTECT
+	flags = HEADCOVERSEYES|HEADCOVERSMOUTH|HEADBANGPROTECT
 	armor = list(melee = 41, bullet = 15, laser = 5,energy = 5, bomb = 5, bio = 2, rad = 0)
 	flags_inv = HIDEMASK|HIDEEARS|HIDEEYES|HIDEFACE
 	strip_delay = 80
 	action_button_name = "Toggle Helmet Visor"
+	visor_flags = HEADCOVERSEYES|HEADCOVERSMOUTH
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: this one is probably also wrong

File: code/modules/clothing/head/helmet.dm [hunk 8]
```diff
@@ -51,14 +37,14 @@
 	toggle_message = "You pull the visor down on"
 	alt_toggle_message = "You push the visor up on"
 	can_toggle = 1
-	flags = HEADBANGPROTECT
+	flags = HEADCOVERSEYES|HEADCOVERSMOUTH|HEADBANGPROTECT
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: wrong

File: code/modules/clothing/head/helmet.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/clothing/head/helmet.dm]: something fucky is going on hold on

**Comment by optimumtact** [comment]
Thats what you get for using git commit -a

sorry @RemieRichards

**Comment by RemieRichards** [comment]
rofl.

File: code/modules/clothing/head/helmet.dm [hunk 9]
```diff
@@ -12,28 +12,15 @@
 	max_heat_protection_temperature = HELMET_MAX_TEMP_PROTECT
 	strip_delay = 60
 	burn_state = -1 //Won't burn in fires
-	var/obj/machinery/camera/portable/helmetCam = null
-	var/spawnWithHelmetCam = 0
-	var/canAttachCam = 0
-	flags_cover = HEADCOVERSEYES
```
Comment (RemieRichards) [inline code/modules/clothing/head/helmet.dm]: this one is still kill @optimumtact

**Comment by palpatine213** [comment]
Why remove them? Just remove them spawning on sec officers round start.

**Comment by optimumtact** [comment]
Because consensus indicated that without roundstart spawn of this feature the feature would never be used and the code would rot over time

File: code/modules/clothing/head/helmet.dm (hunk 9 above)
Comment (optimumtact) [inline code/modules/clothing/head/helmet.dm]: ta

**Comment by palpatine213** [comment]
Alright then. makes sense

**Comment by Falamazeer** [comment]
AI backed security is the best they got, between this, the armor nerfs, the slow shots, and the no doubt soon to be ganked flash you are de-clawing a department that already needs help.

We'll be back to low-pop soon, Please reconsider this from a balance perspective, and limit rather than remove, even single tile camera's could be helpful as it would allow AI to quickly locate and examine them and more quickly aid in a rescue attempt in maintenance without revealing the name or location of the attacker.

**Comment by Kromgar** [comment]
Ahahaha Falamazeer people want a more lethal ss13

**Comment by optimumtact** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=3247&start=50

**Comment by optimumtact** [comment]
How does only seeing 1 tile differ from suit sensors?

**Comment by optimumtact** [comment]
https://tgstation13.org/phpBB/viewtopic.php?f=10&t=4022

Please leave thoughts in this thread

**Comment by optimumtact** [comment]
Yo it's ready

most people hate this thing

**Comment by Supermichael777** [comment]
I like it mere 2/5 of the poll liked it this is the maximum of I died pls nerf anti sec bullshit. just because a narrow majority of fourmites didn't like it it's getting outright pulled? this isn't wasent and shouldn't be how tg station works. we didn't do it for goof chem we shouldn't do it for head cam. this haven't even provoked the same vocal hate just dislike form some coders and fourmites. :-1:

**Comment by optimumtact** [comment]
There's always bay if you want an easier time

**Comment by Supermichael777** [comment]
I'm making a controversial change to outright remove a feature based on a false premise that everyone hates it when the pole was 50 - 50. if you don't like it get off my server. I feel we have seen iterActions of this song and dance before. be a good friend, warn others befor they devolve into papareka

**Comment by Strathcona** [comment]
The new crew console changes do kind of make the helmet cams obsolete from a location finding perspective. They also lack the ability to identify the attacker so it's still a nerf. I'd be okay with this.

**Comment by optimumtact** [comment]
The poll was 39, 61 - how disconnected from reality are you. I'm saying if you don't like the choice the community made, then you may always try other communities who's views are more in line with your own.

**Comment by Supermichael777** [comment]
it would be nice if they started with flashlights attached instead. oh well I'm sure these will be back eventually. sorry if I seem antagonistic but I really think just straight removal is a bad idea. sec us going to be interesting when I get back from my trip. I actually think I did get less action with the cam on.

**Comment by Cheridan** [comment]
These were voted to get removed by the community, so here goes.

**Pull request merged**